lib/API/DX/Device.cpp

```diff
@@ -633,16 +649,33 @@ class DXDevice : public offloadtest::Device {
     static uint64_t FenceCounter = 0;
     uint64_t CurrentCounter = FenceCounter + 1;
 
-    if (auto Err = HR::toError(IS.Queue->Signal(IS.Fence, CurrentCounter),
+    if (auto Err = HR::toError(IS.Queue->Signal(IS.Fence.Get(), CurrentCounter),
                                "Failed to add signal."))
       return Err;
 
     if (IS.Fence->GetCompletedValue() < CurrentCounter) {
+#ifdef _WIN32
+      HANDLE Event = IS.Event;
+#else // WSL
+      HANDLE Event = reinterpret_cast<HANDLE>(IS.Event);
+#endif
       if (auto Err = HR::toError(
-              IS.Fence->SetEventOnCompletion(CurrentCounter, IS.Event),
+              IS.Fence->SetEventOnCompletion(CurrentCounter, Event),
               "Failed to register end event."))
         return Err;
+
+#ifdef _WIN32
       WaitForSingleObject(IS.Event, INFINITE);
+#else // WSL
+      pollfd pfd;
```

Comment:
Please make variable names match LLVM coding standards: https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly